alignment_comment.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+#Description:
+#Script aligns fastq files in zipped format with HISAT2
+#Converts Sam output into BAM
+#Then creates BAM Index Files for visualization (.bai)
+
+#Written & Maintained by Vasco Morais (April 2019)
+
+echo $RNA_HOME
+
+export RNA_PRACTICE_DATA_DIR=$RNA_HOME/practice
+echo $RNA_PRACTICE_DATA_DIR
+
+RNA_PATH=$RNA_HOME/practice
+ADAPTER=${RNA_REFS_DIR}'/illumina_multiplex.fa'
+MIX='_r'
+OUTPUT='_Build37-ErccTranscripts-chr22'
+
+echo $RNA_PATH
+echo $ADAPTER
+
+
+echo '-------------------------------'
```

Comment:
Consider making this into a function in a separate file like `utils.sh` instead of repeating it here. You can just import that anywhere
```
printRuler () { 
 echo '-------------------------------'
}
```